code/modules/mob/living/carbon/life.dm

```diff
@@ -174,70 +171,6 @@
 	handle_stagger()
 	handle_disabilities()
 
-/mob/living/carbon/proc/breathe()
-	if(!need_breathe())
-		return
-
-	if(pulledby && pulledby.grab_state >= GRAB_KILL)
-		Losebreath(3)
-
-	if(health < get_crit_threshold() && !reagents.has_reagent(/datum/reagent/medicine/inaprovaline))
-		Losebreath(1, TRUE)
```

Comment:
This and the lines following is the code that makes it so mobs in crit suffocate and die. I'd recommend leaving in breathe(), need_breathe() to handle stasis stuff, and handle_breathe changed to just run off a bool as to whether you should heal or take damage that breath. It's also going to kill needing a mask for ice maps and anesthetic tanks working on surgical beds.